src/server/seeds/development/02_channels.js

```diff
@@ -7,73 +7,90 @@ exports.seed = function(knex) {
       return knex('channels').insert([
         {
           id: 1,
-          title: 'Front Desk',
+          title: 'Kitchen',
           created_at: '2020-10-01 12:00:00',
           updated_at: '2020-10-01 12:00:00',
           deleted_at: '2020-10-01 12:00:00',
+          imageUrl:
+            'https://upload.wikimedia.org/wikipedia/commons/c/c5/Kitchen_jantje_zag_eens_pruimen_hangen.jpg',
         },
         {
           id: 2,
-          title: ' kitchen',
+          title: 'FoH',
           created_at: '2020-10-01 12:10:00',
           updated_at: '2020-10-01 12:10:00',
           deleted_at: '2020-10-01 12:10:00',
+          imageUrl:
+            'https://upload.wikimedia.org/wikipedia/commons/thumb/0/0a/Cyclonebill_Hvide_asparges_med_pocheret_aeggeblomme_og_skovmaerkesauce.jpg/220px-Cyclonebill_Hvide_asparges_med_pocheret_aeggeblomme_og_skovmaerkesauce.jpg',
         },
         {
           id: 3,
-          title: null,
+          title: 'Maintenance',
           created_at: '2020-10-01 12:20:00',
           updated_at: '2020-10-01 12:20:00',
           deleted_at: '2020-10-01 12:20:00',
+          imageUrl:
+            'https://upload.wikimedia.org/wikipedia/commons/thumb/6/62/Barbieri_-_ViaSophia25668.jpg/435px-Barbieri_-_ViaSophia25668.jpg',
         },
         {
           id: 4,
-          title: 'Hotels',
+          title: 'Delivery',
           created_at: '2020-10-01 12:30:00',
           updated_at: '2020-10-01 12:30:00',
           deleted_at: '2020-10-01 12:30:00',
+          imageUrl:
+            'https://upload.wikimedia.org/wikipedia/commons/thumb/4/42/UberEats_cyclist_in_Amsterdam.jpg/330px-UberEats_cyclist_in_Amsterdam.jpg',
         },
         {
           id: 5,
-          title: 'Employees',
+          title: 'null',
           created_at: '2020-10-02 12:20:00',
           updated_at: '2020-10-02 12:20:00',
           deleted_at: '2020-10-02 12:20:00',
+          imageUrl: 'null',
         },
         {
           id: 6,
-          title: null,
+          title: 'null',
```

Comment:
Please use something else here instead of `'null'`. For example `'chat'` is better.